### PR TITLE
[Fix](partition) Skip rowset partition id eq 0 smaller than config wh…

### PR DESCRIPTION
**Pull request by deardeng**
…en be start

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by deardeng** [comment]
run buildall

**Comment by deardeng** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

File: be/src/olap/data_dir.cpp [hunk 1]
```diff
@@ -486,6 +505,13 @@ Status DataDir::load() {
             continue;
         }
 
+        if (rowset_meta->partition_id() == 0) {
+            LOG(WARNING) << "skip tablet_id=" << tablet->tablet_id()
```
Comment (yiguolei) [inline be/src/olap/data_dir.cpp:509]: If there is a committed rowset with partition id = 0; and you ignore it. Then publish version will success and FE will finish the txn. But BE will not have this data, then data loss and query will failed

**Comment by deardeng** [comment]
run buildall

**Comment by deardeng** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Comment by deardeng** [comment]
run buildall

**Pull request merged**